Automattic/wp-calypso
# Create route & component to handle Jetpack Simple Checkout Calendly scheduling

#### Changes proposed in this Pull Request

* New route for Scheduling Jetpack Onboarding, `/checkout/jetpack/schedule-happiness-appointment`
   * requires user logged in
   * use new route on site less thank-you page 
* New Component to handle scheduling with embedded Calendly Widget styled with Jetpack Main Color

#### Testing instructions

1. Set `jetpack/siteless-checkout` to true in `development.json` and boot branch locally
2. Navigate to `/checkout/jetpack/thank-you/no-site/jetpack_scan` as a logged in user
2. Click "schedule a 15 min call now." in the lower right
3. Verify you are sent to `/checkout/jetpack/schedule-happiness-appointment`, matching the screenshot below: <img width="1409" alt="Screen Shot 2021-07-14 at 11 59 31 AM" src="https://user-images.githubusercontent.com/2810519/125681245-2fe17ca8-d7a2-4550-af5b-44cdb3951dc1.png">
4. Complete Scheduling an appointment, verifying that your user info is pre-filled<img width="1409" alt="Screen Shot 2021-07-14 at 11 59 37 AM" src="https://user-images.githubusercontent.com/2810519/125681187-41998b7d-2f1d-4fc8-82b8-2c07ba04572b.png">
5. Verify you are sent an email scheduling your appointment


related to: 1200479326344990-as-1200537578270027
